airflow/api_connexion/schemas/dataset_schema.py

```diff
@@ -91,6 +92,25 @@ class DatasetCollectionSchema(Schema):
 dataset_collection_schema = DatasetCollectionSchema()
 
 
+class BasicDAGRunSchema(SQLAlchemySchema):
+    """Basic Schema for DAGRun"""
+
+    class Meta:
+        """Meta"""
+
+        model = DagRun
+        dateformat = "iso"
+
+    run_id = auto_field(data_key='dag_run_id')
+    dag_id = auto_field(dump_only=True)
+    execution_date = auto_field(data_key="logical_date", dump_only=True)
```

Comment:
execution_date is only included for compat reasons, let's not include it in a new API.

```suggestion
    logical_date = auto_field(dump_only=True)
```

Reply:
the dag run model still uses `execution_date` but the `data_key`, changes the name to logical date for the API docs.